Readme.md

```diff
@@ -1,6 +1,10 @@
 # 1-Click Upgrade
 
 [![Build Status](https://travis-ci.com/PrestaShop/autoupgrade.svg?branch=dev)](https://travis-ci.com/PrestaShop/autoupgrade)
```

Comment:
Travis has been removed

Reply:
Fixed in https://github.com/PrestaShop/autoupgrade/pull/382/